gfx_window.h

```diff
@@ -44,7 +44,8 @@ enum GfxCreateWindowFlag
     kGfxCreateWindowFlag_NoResizeWindow   = 1 << 1,
     kGfxCreateWindowFlag_HideWindow       = 1 << 2,
     kGfxCreateWindowFlag_FullscreenWindow = 1 << 3,
-    kGfxCreateWindowFlag_AcceptDrop       = 1 << 4
+    kGfxCreateWindowFlag_AcceptDrop       = 1 << 4,
+    kGfxCreateWindowFlag_FitToScreen      = 1 << 5
```

Comment:
New user option to shrink resolution to enable the window to fit entirely on screen. I made this an option as users may want keep the back buffer resolution at the specified size (for benchmarking, render to texture etc.)